.github/actions/build-zui/action.yml

```diff
@@ -87,5 +87,5 @@ runs:
     - name: Check notorization with gatekeeper
       if: runner.os == 'macOS'
       run: |
-        spctl --assess --type execute --verbose --ignore-cache --no-cache dist/apps/zui/mac/*.app
+        spctl --assess --type execute --verbose --ignore-cache --no-cache dist/apps/zui/mac*/*.app
```

Comment:
The wildcard is needed here because the Apple Silicon builds end up in a subdirectory `mac-arm64/` whereas the Intel ones were in `mac/`.